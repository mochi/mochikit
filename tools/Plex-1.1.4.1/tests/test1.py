import Test
from Plex import *
import sys

lex = Lexicon([
  (Any("ab") + Rep(Any("ab01")), 'ident'),
  (Any(" \n"), IGNORE)
],
debug = Test.debug,
timings = sys.stderr
)

Test.run(lex, "test1", debug = 0, trace = 0)


