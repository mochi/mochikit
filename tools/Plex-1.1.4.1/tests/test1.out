test1.in   1   0 ident      'a'
test1.in   1   2 ident      'b'
test1.in   1   4 ident      'aa'
test1.in   1   7 ident      'ab'
test1.in   1  10 ident      'ba'
test1.in   1  13 ident      'a0'
test1.in   1  16 ident      'a1'
test1.in   1  19 ident      'b0'
test1.in   1  22 ident      'b1'
test1.in   1  25 ident      'ab01'
test1.in   2   0 ident      'babba01'
test1.in   2   8 ident      'abba'
test1.in   2  13 ident      'babb'
test1.in   2  18 ident      'b0001a'
test1.in   4   0 None       ''
